nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -333,6 +340,18 @@ class MatplotlibSurfaceBackend(BaseSurfaceBackend):
     def name(self):
         return "matplotlib"
 
+    def _check_figure_axes_inputs(self, figure, axes):
+        """Check if the specified figure and axes are matplotlib objects."""
+        if figure is not None and not isinstance(figure, plt.Figure):
+            raise ValueError(
+                "figure argument should be None or a "
+                "'matplotlib.pyplot.Figure'."
+            )
+        if axes is not None and not isinstance(axes, plt.Axes):
+            raise ValueError(
+                "axes argument should be None or a 'matplotlib.pyplot.Axes'."
+            )
+
```

Comment:
This function is moved from `surf_plotting` to here changing the contents. It was checking if the specified figure/axes were `PlotlySurfaceFigure` or not which actually does not cover valid values. I've changed comparison to None or matplotlib objects.